habitat_sim/robots/mobile_manipulator.py

```diff
@@ -378,13 +414,35 @@ def arm_joint_pos(self, ctrl: List[float]):
             joint_positions[self.joint_pos_indices[jidx]] = ctrl[i]
         self.sim_obj.joint_positions = joint_positions
 
-    def _validate_arm_ctrl_input(self, ctrl: List[float]):
+    @property
+    def leg_joint_pos(self) -> np.ndarray:
+        """Get the current arm joint positions."""
+        leg_pos_indices = list(
+            map(lambda x: self.joint_pos_indices[x], self.params.leg_joints)
```

Comment:
No need for map and list. This can just be a generator, it's only iterated through once.